components/ide/jetbrains/image/gha-update-image/update-versions.ts

```diff
@@ -0,0 +1,105 @@
+import { $ } from "bun";
+import yaml from "yaml";
+import { z } from "zod";
+
+$.nothrow(); // git likes to respond with non-zero codes, but it is alright for us
+
+const versionManifest = z.object({
+    components: z.object({
+        workspace: z.object({
+            desktopIdeImages: z.record(z.string(), z.object({ version: z.string() })),
+        }),
+    }),
+});
+
+const workspaceYaml = z.object({
+    defaultArgs: z.object({
+        intellijDownloadUrl: z.string(),
+        golandDownloadUrl: z.string(),
+        pycharmDownloadUrl: z.string(),
+        phpstormDownloadUrl: z.string(),
+        rubymineDownloadUrl: z.string(),
+        webstormDownloadUrl: z.string(),
+        riderDownloadUrl: z.string(),
+        clionDownloadUrl: z.string(),
+    }),
+});
+
+const tagInfo =
```

Comment:
These three waitings may fail, in this case, it only logs zod parse failed. Could we log something to help us understand what's going on if it failed?

Reply:
Adding some more error messages for those cases